starcheck/src/starcheck.pl

```diff
@@ -587,6 +628,11 @@ sub json_obsids{
 }
 
 
+# Set the thermal model run start time to be either the supplied
+# run_start_time or now.
+my $run_start_time = run_start_time($par{run_start_time}, $bs[0]->{date}) || get_now_time();
```

Comment:
The `get_run_start_time` function should do the entire job:
```
my $run_start_time = get_run_start_time($par{run_start_time}, $bs[0]->{date});
```
Then you don't really need the new `get_now_time()` function.